File: Lv.1/두 정수 사이의 합.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    
    int c=min(a,b);
    int d=max(a,b);
    
    
    for(int i=c;i<=d;i++)
    {
        answer+=i;
    }
    return answer;
}

File: Java/LV1/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        
        long min = (long)Math.min(a,b);
        long max = (long)Math.max(a,b);
        for(long i=min;i<=max;i++){
            answer+=i;
        }
        return answer;
    }
}
